docs/core/diagnostics/event-counter-perf.md

```diff
@@ -0,0 +1,100 @@
+{
+    // Define the singleton instance of the event source
+    public static MinimalEventCounterSource Log = new MinimalEventCounterSource();
+    private EventCounter RequestCounter;
+
+    private MinimalEventCounterSource() : base(EventSourceSettings.EtwSelfDescribingEventFormat)
+    {
+        RequestCounter = new EventCounter("request", this);
+    }
+
+    // Indicate that a request for a URL was made which took a particular amount of time
+    public void Request(string url, float elapsedMSec)
+    {
+        // Notes:
+        //   1. the event ID passed to WriteEvent (1) corresponds to the (implied) event ID
+        //      assigned to this method. The event ID could have been explicitly declared
+        //      using an EventAttribute for this method
+        //   2. Each counter supports a single float value, so conceptually it maps to a single
+        //      measurement in the code.
+        //   3. You don't have to have log with WriteEvent if you don't think you will ever care about details
+        //       of individual requests (that counter data is sufficient).
+
+        // This logs it to the event stream if events are on.
+        WriteEvent(1, url, elapsedMSec);
+        // This adds it to the PerfCounter called 'Request' if PerfCounters are on
+        RequestCounter.WriteMetric(elapsedMSec);
+    }
+}
+```
+
+The <xref:System.Diagnostics.Tracing.EventSource.WriteEvent%2A?displayProperty=nameWithType> line is the <xref:System.Diagnostics.Tracing.EventSource> part and is not part of <xref:System.Diagnostics.Tracing.EventCounter>, it was written to show that you can log a message together with the event counter.
+
+You logged the metric to the <xref:System.Diagnostics.Tracing.EventCounter>, but unless you can actually get the statistics out of it, it is not useful. To get the statistics, you need to enable the <xref:System.Diagnostics.Tracing.EventCounter> by setting off a timer how frequently you want the events, as well as a listener to capture the events, to do that, you can use PerfView.
+
+There is an extra keyword that you will need to specify the turn on the EventCounters.
+
+```
+PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:EventCounterIntervalSec=1 collect
+```
+
+> [!IMPORTANT]
+> The `EventCounterIntervalSec` segment is used to indicate the frequency of the sampling.
+
+As usual, turn on PerfView, and then run the sample code - we get have something like this.
+
+:::image type="content" source="media/perfview-counters.png" lightbox="media/perfview-counters.png" alt-text="PerfView of EventCounter traces":::
+
+Now let's drill into what the data captured means - when I copied from PerfView, it looks like this
+
+```
+ThreadID="17,800" ProcessorNumber="5" Payload="{ Name:"request", Mean:142.0735, StandardDeviation:42.07355, Count:2, Min:100, Max:184.1471, IntervalSec:1.000588 }"
+```
+
+Now it is obvious that within a sampling period, we have nine events, and all the other statistics.
```

Comment:
```suggestion
Within a sampling period, there are nine events and all the other statistics.
```

I don't see nine events, I do see 7 properties in the payload -- is that "all the other statistics"?